src/main/java/uk/gov/cshr/civilservant/controller/CivilServantController.java

```diff
@@ -39,18 +44,22 @@
 
     private final CivilServantResourceFactory civilServantResourceFactory;
 
+    private final OrganisationalUnitRepository organisationalUnitRepository;
+
     public CivilServantController(LineManagerService lineManagerService, CivilServantRepository civilServantRepository,
                                   RepositoryEntityLinks repositoryEntityLinks,
-                                  CivilServantResourceFactory civilServantResourceFactory) {
+                                  CivilServantResourceFactory civilServantResourceFactory,
+                                  OrganisationalUnitRepository organisationalUnitRepository) {
         this.lineManagerService = lineManagerService;
         this.civilServantRepository = civilServantRepository;
         this.repositoryEntityLinks = repositoryEntityLinks;
         this.civilServantResourceFactory = civilServantResourceFactory;
+        this.organisationalUnitRepository = organisationalUnitRepository;
```

Comment:
See comment below, i dont think we should be calling the organisationalUnitRepository directly from the CivilServantController.

Reply:
See above